_index.md

```diff
@@ -67,6 +67,7 @@ The table lists information from vendors related to support for Post Quantum Cry
 | [Keyfactor](#keyfactor)                 | SignServer       | Signing Software | 2022-12-19   |                  :x:                  |                :x:                 |        :heavy_check_mark:         |         :x:         |          :x:          |    :heavy_check_mark:    |   :heavy_check_mark:    |         :x:          |         :x:         |           :x:           |        :x:         |
 | [Keyfactor](#keyfactor)                 | EJBCA            | PKI              | 2023-11-20   |                  :x:                  |                :clock1:          |        :x:         |         :x:         |  :heavy_check_mark:   |    :heavy_check_mark:    |           :x:           |         :x:          |         :x:         |           :x:           |        :x:         |
 | [MTG AG](#mtg-ag)                       | Corporate PKI    | PKI              | 2023-09-25   |                  :x:                  |                :x:                 |        :x:         |         :x:         |  :heavy_check_mark:   |    :heavy_check_mark:    |   :heavy_check_mark:    |       :clock1:       |         :x:         |   :heavy_check_mark:    |        :x:         |
+| [Nexus Group (IN Groupe)](#nexus-group-in-groupe)                       | Certificate Manager    | PKI              | 2024-03-13   |                  :x:                  |                :x:                 |        :x:         |         :x:         |  :clock1:   |    :clock1:    |   :clock1:    |       :x:       |         :x:         |   :x:    |        :x:         |
```

Comment:
Do you need to state the ownership structure (IN Groupe)? It's well visible on Nexus's web page.

Reply:
I made an additional commit to propose an updated version.